src/index.js

```diff
@@ -12,3 +12,7 @@ const loadApp = async () => {
 const loadOauthRedirect = () => import('src/oauthRedirectLoader');
 
 window.location.pathname.startsWith('/redirect-from-oauth') ? loadOauthRedirect() : loadApp();
+
+// This is needed for libraries that assume access to global, as vite does not define globla.
```

Comment:
nit: spelling globla ==> global

Reply:
Well, it also doesn't define `globla`. :)

Reply:
Fixed